sdk/communication/azure-communication-sms/README.md

```diff
@@ -94,32 +74,62 @@ SmsClient smsClient = new SmsClientBuilder()
 
 ## Examples
 
-### Sending a message
-Use the `sendMessage` function to send a new message to a list of phone numbers.
-(Currently SMS Services only supports one phone number).
+### Sending a message to a single recipient
+Use the `send` function to send a new message to a phone number.
 Once you send the message, you'll receive a response where you can access several
-properties such as the message id with the `response.getMessageId()` function.
+properties such as the message id with the `messageResponseItem.getMessageId()` function.
 
-<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L43-L59 -->
+<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L59-L75 -->
 ```java
-// Currently Sms services only supports one phone number
-List<PhoneNumberIdentifier> to = new ArrayList<PhoneNumberIdentifier>();
-to.add(new PhoneNumberIdentifier("<to-phone-number>"));
+//Send an sms to only one phone number
+String to = "<to-phone-number>";
 
-// SendSmsOptions is an optional field. It can be used
 // to enable a delivery report to the Azure Event Grid
-SendSmsOptions options = new SendSmsOptions();
-options.setEnableDeliveryReport(true);
-
-// Send the message and check the response for a message id
-SendSmsResponse response = smsClient.sendMessage(
-    new PhoneNumberIdentifier("<leased-phone-number>"),
+SmsSendOptions options = new SmsSendOptions();
+options.setDeliveryReportEnabled(true);
+//addionaly you can ad a tag you wish to identify the messages for this tag.
+options.setTag("Tag"); /* Optional */
+
+// Send the message to a list of  phone Numbers and check the response for a messages ids
+SmsSendResult response = smsClient.send(
+    "<leased-phone-number>",
     to,
     "your message",
     options /* Optional */);
 
 System.out.println("MessageId: " + response.getMessageId());
 ```
+### Sending a message to multiple recipients
+Use the `send` function to send a new message to a list of phone numbers.
+Once you send the message, you'll receive a PagedIterable response where you can access several
+properties such as the message id with the `messageResponseItem.getMessageId()` function.
+
+<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L80-L102 -->
+```java
+//Send an sms to multiple phone numbers
+List<String> toMultiplePhones = new ArrayList<String>();
+toMultiplePhones.add("<to-phone-number1>");
+toMultiplePhones.add("<to-phone-number2>");
```

Comment:
Use fake numbers that have area code as  555. E.g. "+15550000000"

Reply:
actually Dom mentioned in my review that using "<to-phone-number1>" and such is good practice

Reply:
Thanks Bert, yea, seems like we should just keep as is then

Reply:
As per the bug https://github.com/Azure/azure-sdk-for-js/issues/12918 decided to let it as it is